e2e/cloud-plugins-suite/azure-monitor.spec.ts

```diff
@@ -20,10 +20,11 @@ type AzureMonitorConfig = {
 
 type AzureMonitorProvision = { datasources: AzureMonitorConfig[] };
 
-const dataSourceName = `Azure Monitor E2E Tests - ${uuidv4()}`;
+let dataSourceName = '';
```

Comment:
nit: you don't need this change now either :P

Reply:
Oops, thanks!